plugins/modules/system/dconf.py

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# (c) 2017, Branko Majic <branko@majic.rs>
+# Copyright: (c) 2017, Branko Majic <branko@majic.rs>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 module: dconf
 author:
     - "Branko Majic (@azaghal)"
 short_description: Modify and read dconf database
 description:
-  - This module allows modifications and reading of dconf database. The module
-    is implemented as a wrapper around dconf tool. Please see the dconf(1) man
-    page for more details.
+  - This module allows modifications and reading of C(dconf) database.
+  - The module is implemented as a wrapper around C(dconf) tool.
+  - Please see the dconf(1) man page for more details.
```

Comment:
I haven't checked, but this would turn it into three paragraphs, right? Just being pedantic here - not sure if it makes sense to have the three sentences as three paragraphs?

Reply:
Yes, it would be three paragraphs.

Reply:
When I try to read those three sentences as part of their own paragraphs, it kinda feels weird for me. I'd probably at least merge the second and third sentence into same paragraph. But again - just being pedantic here.